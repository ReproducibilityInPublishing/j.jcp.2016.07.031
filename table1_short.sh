echo "AIMGM" > table1_short.csv

./example1 -l 9 -a 0.1666667 -N 8 2>/dev/null >> table1_short.csv
./example1 -l 9 -a 0.1666667 -N 16 2>/dev/null >> table1_short.csv

./example1 -l 9 -a 0.5 -N 32 2>/dev/null >> table1_short.csv
./example1 -l 9 -a 0.5 -N 64 2>/dev/null >> table1_short.csv

./example1 -l 9 -a 0.99 -N 100 2>/dev/null >> table1_short.csv
./example1 -l 9 -a 0.99 -N 200 2>/dev/null >> table1_short.csv

echo "BD-ADI" >> table1_short.csv

./example1_BDADI -l 9 -a 0.1666667 -N 8 2>/dev/null >> table1_short.csv
./example1_BDADI -l 9 -a 0.1666667 -N 16 2>/dev/null >> table1_short.csv

./example1_BDADI -l 9 -a 0.5 -N 32 2>/dev/null >> table1_short.csv
./example1_BDADI -l 9 -a 0.5 -N 64 2>/dev/null >> table1_short.csv

./example1_BDADI -l 9 -a 0.99 -N 100 2>/dev/null >> table1_short.csv
./example1_BDADI -l 9 -a 0.99 -N 200 2>/dev/null >> table1_short.csv
