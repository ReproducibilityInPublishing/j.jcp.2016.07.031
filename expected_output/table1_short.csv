AIMGM
0,0,3.14159,3.14159,0.5,5e-09,1e-08,512,0.166667,8,3.4,1.39465,1.376246e-03
0,0,3.14159,3.14159,0.5,5e-09,1e-08,512,0.166667,16,4,2.93361,3.465207e-04
0,0,3.14159,3.14159,0.5,5e-09,1e-08,512,0.5,32,5,6.96905,1.161483e-03
0,0,3.14159,3.14159,0.5,5e-09,1e-08,512,0.5,64,5,13.5989,4.176068e-04
0,0,3.14159,3.14159,0.5,5e-09,1e-08,512,0.99,100,5.98039,25.1703,5.855795e-03
0,0,3.14159,3.14159,0.5,5e-09,1e-08,512,0.99,200,5.87129,48.9185,2.810468e-03
BD-ADI
0,0,3.14159,3.14159,0.5,5e-09,1e-08,512,0.166667,8,0,0.138173,4.090709e-02
0,0,3.14159,3.14159,0.5,5e-09,1e-08,512,0.166667,16,0,0.298962,1.934541e-02
0,0,3.14159,3.14159,0.5,5e-09,1e-08,512,0.5,32,0,0.749714,6.500271e-04
0,0,3.14159,3.14159,0.5,5e-09,1e-08,512,0.5,64,0,2.51085,2.294960e-04
0,0,3.14159,3.14159,0.5,5e-09,1e-08,512,0.99,100,0,5.8122,5.900656e-03
0,0,3.14159,3.14159,0.5,5e-09,1e-08,512,0.99,200,0,22.1311,2.939535e-03
